stripe2qbo/db/models.py

```diff
@@ -40,3 +41,14 @@ class QBOToken(Base):
     user_id: Mapped[int] = mapped_column(ForeignKey("users.id"))
 
     user = relationship("User", back_populates="qbo_token")
+
+
+class TransactionSync(Base):
```

Comment:
Looks good, but should be in a different PR. Ideally, each PR does one thing, which can be understood from the description.

In this case, I read the title "Added dummy test", and this is completely unrelated